dockerfiles/ci/buster/php-5.4/Dockerfile

```diff
@@ -86,7 +112,7 @@ RUN set -eux; \
     switch-php debug;
 
 # Install Composer
-COPY --from=composer:2 /usr/bin/composer /usr/local/bin/composer
+COPY --from=composer:1 /usr/bin/composer /usr/local/bin/composer
```

Comment:
Composer 2 has issues on php 5 that I have not been able to sort out in the restricted time at my disposal. See https://github.com/DataDog/dd-trace-php/pull/1174